### PR TITLE
[AC-659] Add Contact Us Screen. 

### PR DESCRIPTION
**Pull request by ribhavsharma**
## Description of what I changed

Added a Contact Us Screen that sends an email to the OpenMRS HelpDesk.  
- [x] Added it in the overflow menu.
- [x] Added it in the settings fragment.

## Screenshot
Screenshot_20200121-144846 (2)

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-659

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 package org.openmrs.mobile.activities.settings;
 
+import static java.security.AccessController.getContext;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java]: Is this import necessary?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java [hunk 2]
```diff
@@ -202,6 +207,14 @@ public void setDarkMode() {
...
     }
 
+    @Override
+    public void setUpcontactUsPage() {
+        LinearLayout contactUsLL = root.findViewById(R.id.frag_settings_contact_us_ll);
+        contactUsLL.setOnClickListener(v -> {
+            Intent contactIntent = new Intent(getActivity(), ContactUsActivity.class);
+            startActivity(contactIntent);
+        });
+    }
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java]: Add a new line here.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+        package org.openmrs.mobile.activities;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: This doesn't have correct indentation.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import com.google.android.material.button.MaterialButton;
+import com.google.android.material.textfield.TextInputEditText;
+import org.openmrs.mobile.R;
+
+
+public class ContactUsActivity extends ACBaseActivity {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: This class should follow the MVP design pattern. Use [the settings screen](https://github.com/openmrs/openmrs-contrib-android-client/tree/master/openmrs-client/src/main/java/org/openmrs/mobile/activities/settings) as an example. There should be a `ContactUsContract`, `ContactUsPresenter`, `ContactUsActivity` and `ContactUsFragment`.

Once you're done, you should test the email sent, check if the intent is an email intent and that the email body is valid.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    private MaterialButton sendButton, forumButton, ircButton;
+    private TextInputEditText nameTIET, subjectTIET, messageTIET;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_contact_us);
+        initViewFields();
+        sendButton.setOnClickListener(view -> {
+            String name_str = nameTIET.getText().toString();
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: Variable names should be `camelCase`

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+            String subject_str = subjectTIET.getText().toString();
+            String msg_str = messageTIET.getText().toString();
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: See above comment

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java [hunk 7]
```diff
@@ -41,6 +41,8 @@
 import androidx.localbroadcastmanager.content.LocalBroadcastManager;
 import org.openmrs.mobile.R;
 import org.openmrs.mobile.activities.ACBaseFragment;
+import org.openmrs.mobile.activities.contactus.ContactUsActivity;
+import org.openmrs.mobile.activities.contactus.ContactUsActivity;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java]: This is a duplicate import.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/contactus/ContactUsFragment.java [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+    public static ContactUsFragment newInstance() {
+        return new ContactUsFragment();
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        root = inflater.inflate(R.layout.fragment_contact_us, container, false);
+        initViewFields();
+        String name_str = nameTiet.getText().toString();
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/contactus/ContactUsFragment.java]: These 3 variables should be `camelCase`.

**Comment by f4ww4z** [comment]
Closing due to inactivity. @rishabh-997 Will you be able to take this up? You can see this PR's code for reference.

**Comment by rishabh-997** [comment]
> Closing due to inactivity. @rishabh-997 Will you be able to take this up? You can see this PR's code for reference.

will be ready by tonight

**Pull request closed without merging**